src/components/Conversion/ConversionNOB.vue

```diff
@@ -80,57 +80,50 @@
 <script lang="ts">
 import { Action, Getter } from 'vuex-class'
 import { Component, Vue, Emit, Watch } from 'vue-property-decorator'
-import { ActionBindingIF, EntitySnapshotIF } from '@/interfaces/'
+import { ActionBindingIF } from '@/interfaces/'
 import { NaicsIF } from '@bcrs-shared-components/interfaces/'
 import { isEqual } from 'lodash'
 
 @Component({})
 export default class NatureOfBusiness extends Vue {
   @Getter getCurrentNaics!: NaicsIF
-  @Getter getEntitySnapshot!: EntitySnapshotIF
+  @Getter getSnapshotNaics!: NaicsIF
   @Getter hasNatureOfBusinessChanged!: boolean
 
   @Action setNaics!: ActionBindingIF
   @Action setValidComponent!: ActionBindingIF
 
   // local variables
-  private dropdown: boolean = null
-  private onEditMode = false
-  private naicsText = ''
-  private naicsRules = [
+  protected dropdown = false
+  protected onEditMode = false
+  protected naicsText = ''
+  protected naicsRules = [
```

Comment:
Because these are used in the template, they can't be private. (It still works with private but I get VC Code warnings.)